src/test/k3s/test/keycloak/service.yaml

```diff
@@ -7,6 +7,9 @@ spec:
     - name: http
       port: 80
       targetPort: http
+    - name: cache
```

Comment:
will tghis be used with istio?
port name prefix might be helpful: https://istio.io/v1.0/docs/setup/kubernetes/spec-requirements/#:~:text=The%20port%20names%20must%20be,but%20name%3A%20http2foo%20is%20not.